packages/@webex/internal-plugin-device/test/unit/spec/device.js

```diff
@@ -210,6 +210,63 @@ describe('plugin-device', () => {
       });
     });
 
+    describe('#register()', () => {
+      let requestSpy;
+
+      const setup = () => {
+        webex.internal.metrics.submitClientMetrics = sinon.stub();
+        sinon.stub(device, 'canRegister').callsFake(() => Promise.resolve());
+        sinon.stub(device, 'processRegistrationSuccess').callsFake(() => {});
+
+        device.config.defaults = {};
+        device.set('registered', false);
+      };
+
+      it('checks that submitInternalEvent gets called with internal.register.device.request', async () => {
+        setup();
+        requestSpy = sinon.spy(device, 'request');
```

Comment:
I don't see requestSpy being used anywhere, were you planning on checking that it is being called?

Reply:
Blame working in office ;D